src/aura/PostStatusPath/PostStatusPathHelper.js

```diff
@@ -0,0 +1,55 @@
+({
+	updatePath: function (component) {
+		var currentStatus = component.get("v.current");
+		var li = component.find(currentStatus);
```

Comment:
Is important to choose intuitive names for variables, functions and components in general, those names need to point out briefly the purpose of a variable or component in the app

Reply:
Ok, I set this name because I adding css class to a `li` tag with the current status, but you right it's be need more intuitive.